saleor/graphql/product/mutations/product_variant/product_variant_create.py

```diff
@@ -294,9 +296,18 @@ def check_for_duplicates_in_stocks(cls, stocks_data):
                 }
             )
 
+    @classmethod
+    def set_track_inventory(cls, _info, instance, cleaned_input):
+        instance.track_inventory = (
+            get_track_inventory_by_default()
```

Comment:
After recent changes, `get_track_inventory_by_default` can return `None`, an invalid value for a boolean field. We would have to rewrite this part to ensure that we set `instance.track_inventory` when we get a non-null result from `get_track_inventory_by_default`. This applies to all usages of this function.

Reply:
Yeah that's why I raised error. NVM now added code to handle None.